usaspending_api/search/v2/views/spending_over_time.py

```diff
@@ -95,11 +98,35 @@ def post(self, request):
 
         db_results, values = self.database_data_layer()
 
-        # build response
-        for result in db_results:
-            result["time_period"] = {}
-            for period in values:
-                result["time_period"][self.groupings[period]] = str(result[period])
-                del result[period]
-
-        return Response({"group": self.groupings[self.group], "results": db_results})
+        # Populate all possible periods with no new awards for now
+        hashed_results = {}
+
+        # time_period is optional so we're setting a default window from API_SEARCH_MIN_DATE to today.
+        # Otherwise, users will see blank results for years
+        default_time_period = {'start_date': settings.API_SEARCH_MIN_DATE,
+                               'end_date': datetime.utcnow().strftime('%Y-%m-%d')}
```

Comment:
For spending_over_time, I recommend setting the default to either the end of the current FY or the API_MAX_DATE instead of the current day due to the different materialized views aggregating by different date periods.